app/src/components/market/common/market_resolution_message/index.tsx

```diff
@@ -85,39 +89,70 @@ const MarketResolutionMessage = (props: Props) => {
   if (invalid) {
     infoTitle = 'Market is invalid!'
     icon = <Invalid />
-    infoText = `${arbitrator.name} declared this market invalid. You own ${shares} Shares of ${
+    infoText = `${
+      arbitrator.name
+    } declared this market invalid. You can now redeem redeemString from your ${shares} Shares of ${
       userWinningOutcomes === 1 ? `an outcome` : `${userWinningOutcomes} outcomes`
-    }
-      which can be redeemed for `
+    }`
   }
 
   if (lost) {
     infoTitle = 'Better luck next time!'
     icon = <Lost />
-    infoText = `You have bought ${shares} Shares of the loosing outcome which leads to a loss of `
+    infoText = `You have bought ${shares} Shares of the loosing outcome which leads to a loss of redeemString`
   }
 
   if (wonSingle) {
     infoTitle = 'Congratulations!'
     icon = <Won />
-    infoText = `You have bought ${shares} Shares of the winning outcome which can be redeemed for `
+    infoText = `You can now redeem redeemString from your ${shares} Shares of the winning outcome`
```

Comment:
yeah theres bunch of these redeemString throughout the code am I missing something?

Reply:
Yeah is all good, we replace these later but I should make it less confusing, something like `replace_with_redeem`